why_choose_vinta.md

```diff
@@ -0,0 +1,13 @@
+Bill Gates once mentioned that: **“Your most unhappy customers are your greatest source of learning.”** Sometimes that points you to focusing more on the client’s needs, other times it warns you that you might be looking at the wrong customer. Here we describe the things Vinta looks for as the right customer type. Clients like these are the ones Vinta can deliver most value to. These aspects came from our experience in aligning business value with Vinta team’s values.
+
+## The Aspects We Look For
+* Has work for at least 2 devs -> Developers demands collaboration. We don't believe in lone heroes. We believe in teams. Developers need to validate their work with each other, share experiences, discuss concerns and find ways to improve. Interactions are what make the team greater than the sum of its parts.
+* At least 3 months -> Doing small projects over and over can make developers really good in setups and basic features, but larger projects have more interesting challenges that help developers become experts.
+* No part-time -> Context-switch is horrible, it’s bad for morale and kills productivity. We avoid that. Focus and ownership is much easier to achieve with full-time engagement.
+* Freedom to implement the process that works -> We want, primarily, to deliver end-to-end quality for software products. We built a software development process to guarantee that. And we get that the client’s context might emphasize other concerns. Far from saying that our process is immutable and that we can’t adapt it to better fit the client’s needs. We're happy to add or change some aspects of our process, but we'll never neglect what we see as fundamental, such as testing, refactoring, transparent communication, timely feedback, etc.
+* Possibilities for UX/Product improvements -> We care to deliver value beyond code for features. How far we will be allowed to iterate is up to the client, but we value clients that want us to get involved in the product impact of software features.
+* Validated projects, with real users -> Taking chances and building entire products on hunches can lead to idleness and demotivate developers (no one likes to develop things that never see the light of day). We focus on the good feeling that is to see your code being used and solving real problems for real people.
+* See us as their technical team -> We are not only a software shop with a good number of  experienced developers. We want to craft something amazing together with the client. We are not choosing projects where we are invaluable in a month and disposable in the next. We want to help products to last.
```

Comment:
"We want to help products last." I would remove the "to".

Reply:
I worry that this might mean "we want to help products the last" as in, the least. What do you think?

Reply:
HUM. agree.